### PR TITLE
Add a secured label to the CLI tap responses

### PR DESCRIPTION
**Pull request by rmars**
Adds `secured=yes/no` to the `conduit tap` responses. This uses the `conduit_io_control_plane_ns` labels added in #960, but could be changed to use a `meshed` label being returned by the proxy.

```
$ go run cli/main.go tap deploy/web -n emojivoto --kubeconfig ~/.kube/config
req id=0:2150 src=10.1.0.31:36828 dst=10.1.0.29:80 :method=GET :authority=web-svc.emojivoto:80 :path=/api/list secured=no
req id=0:2151 src=10.1.0.29:51590 dst=emoji-7649584cd-cllzj :method=POST :authority=emoji-svc.emojivoto:8080 :path=/emojivoto.v1.EmojiService/ListAll secured=no
rsp id=0:2151 src=10.1.0.29:51590 dst=emoji-7649584cd-cllzj :status=200 latency=1457µs secured=no
end id=0:2151 src=10.1.0.29:51590 dst=emoji-7649584cd-cllzj grpc-status=OK duration=171µs response-length=2161B secured=no
rsp id=0:2150 src=10.1.0.31:36828 dst=10.1.0.29:80 :status=200 latency=5006µs secured=no
req id=0:2222 src=10.1.0.31:37596 dst=10.1.0.30:80 :method=GET :authority=web-svc.emojivoto:80 :path=/api/vote secured=no
req id=0:2223 src=10.1.0.30:45812 dst=emoji-7649584cd-cllzj :method=POST :authority=emoji-svc.emojivoto:8080 :path=/emojivoto.v1.EmojiService/FindByShortcode secured=no
rsp id=0:2223 src=10.1.0.30:45812 dst=emoji-7649584cd-cllzj :status=200 latency=2154µs secured=no
end id=0:2223 src=10.1.0.30:45812 dst=emoji-7649584cd-cllzj grpc-status=OK duration=266µs response-length=21B secured=no
req id=0:2224 src=10.1.0.30:50536 dst=voting-747789c9f6-qtwq5 :method=POST :authority=voting-svc.emojivoto:8080 :path=/emojivoto.v1.VotingService/VotePoop secured=no
rsp id=0:2224 src=10.1.0.30:50536 dst=voting-747789c9f6-qtwq5 :status=200 latency=1936µs secured=no
```

### CODE REVIEW

File: cli/cmd/tap.go [hunk 1]
```diff
@@ -242,7 +242,16 @@ func writeTapEventsToBuffer(tapClient pb.Api_TapByResourceClient, w *tabwriter.W
...
 func renderTapEvent(event *common.TapEvent) string {
 	dst := util.AddressToString(event.GetDestination())
-	dstPod := event.GetDestinationMeta().GetLabels()["pod"]
+	dstLabels := event.GetDestinationMeta().GetLabels()
+	dstPod := dstLabels["pod"]
+	isSecured := "no"
+	controllerNs := dstLabels["conduit_io_control_plane_ns"]
+	controllerDstNs := dstLabels["dst_conduit_io_control_plane_ns"]
+
+	if controllerNs != "" && controllerNs == controllerDstNs {
```
Comment (siggy) [inline cli/cmd/tap.go]: per your PR description, mind adding a `TODO` here (and perhaps reference an issue?) re: adding a `meshed`/`secured`/whatever label to the tap event?

Comment (rmars) [reply cli/cmd/tap.go]: yeah!

Comment (rmars) [reply cli/cmd/tap.go]: I could just switch this to use the 'meshed' label like the web and stat CLI does, that way everything gets converted once that label starts appearing... wdyt?

Comment (siggy) [reply cli/cmd/tap.go]: yeah i think that sounds good, keep web/stat/tap all working the same way.

**Pull request merged**